libs/payments/ui/src/lib/client/components/PaymentMethodManagement/index.tsx

```diff
@@ -0,0 +1,228 @@
+                    'payment-name-placeholder',
+                    {},
+                    'Full Name'
+                  )}
+                  value={fullName}
+                  onChange={(e) => {
+                    setFullName(e.target.value);
+                    setHasFullNameError(!e.target.value);
+                  }}
+                  aria-required
+                />
+              </Form.Control>
+              {hasFullNameError && (
+                <Form.Message asChild>
+                  <Localized id="next-payment-validate-name-error">
+                    <p className="mt-1 text-alert-red" role="alert">
+                      Please enter your name
+                    </p>
+                  </Localized>
+                </Form.Message>
+              )}
+            </Form.Field>
+          </>
+        )}
+        <Form.Field name="payment">
+          <Form.Control asChild>
+            <div className="relative">
+              <PaymentElement
+                onChange={handlePaymentElementChange}
+                onReady={handleReady}
+                options={{
+                  layout: {
+                    type: 'accordion',
+                    defaultCollapsed: false,
+                    radios: false,
+                    spacedAccordionItems: true,
+                  },
+                }}
+              />
+            </div>
+          </Form.Control>
+
+          {error && (
+            <Form.Message className="text-red-600 text-sm mt-2 pt-2">
+              {error}
+            </Form.Message>
+          )}
+        </Form.Field>
+        {isInputNewCardDetails && (
+          <div className="flex flex-row justify-center pt-4">
+            <Form.Submit asChild>
+              <button
+                type="submit"
+                disabled={
+                  !stripe || !isComplete || isLoading || hasFullNameError
+                }
+                className="w-md h-8 bg-blue-600 text-white py-2 px-4 rounded-md hover:bg-blue-700 disabled:opacity-50 disabled:cursor-not-allowed flex items-center justify-center"
+              >
+                {isLoading ? (
+                  <>
+                    <LoadingSpinner className="w-6 h-6" />
+                  </>
+                ) : (
+                  <Localized id="payment-method-management-save-method">
+                    Save payment info
+                  </Localized>
+                )}
+              </button>
+            </Form.Submit>
+          </div>
+        )}
+        {isNonDefaultCardSelected && !isInputNewCardDetails && (
+          <div className="flex flex-row justify-center pt-4">
+            <Form.Submit asChild>
+              <button
+                type="submit"
+                disabled={!stripe || !isComplete || isLoading}
+                className="w-md h-8 bg-blue-600 text-white py-2 px-4 rounded-md hover:bg-blue-700 disabled:opacity-50 disabled:cursor-not-allowed flex items-center justify-center"
+              >
```

Comment:
issue: the submit button doesn't seem to match the submit button used at checkout.